src/pageLayout/constants/navigationHierarchy.ts

```diff
@@ -194,7 +194,10 @@ export const generateNavItems = (): NavItem[] => {
     },
     {
       id: 'operator',
-      enabled: () => CLOUD && quartzMe?.isOperator === true,
+      enabled: () =>
+        CLOUD &&
+        (identity?.user?.operatorRole === 'read-write' ||
+          identity?.user?.operatorRole === 'read-only'),
```

Comment:
In the future, especially if we have more calls to this expression, we might consider making this a helper function